src/server/rdb_save.cc

```diff
@@ -49,6 +49,7 @@ ABSL_FLAG(dfly::CompressionMode, compression_mode, dfly::CompressionMode::MULTI_
           "set 2 for multi entry zstd compression on df snapshot and single entry on rdb snapshot,"
           "set 3 for multi entry lz4 compression on df snapshot and single entry on rdb snapshot");
 ABSL_FLAG(int, compression_level, 2, "The compression level to use on zstd/lz4 compression");
+ABSL_FLAG(bool, list_rdb_encode_v2, true, "Use v2 rdb encoding of list object");
```

Comment:
can we run the list unit tests once with this true and once with false?
also, didn't you say you wanted the default to be `false` for this? (I could have misunderstood you, it's find to be true if you want it to be such)

Reply:
also^2: I thought you wanted this flag to be toggleable at runtime (i.e. via `CONFIG`), is it not the case?

Reply:
I do not think we need to toggle it - our cloud service does not require redis-6 RDB compatibility.

Reply:
see config_registry.RegisterMutable in server_family you can set the flag in runtime
I wanted the defualt to be true, I dont see a reason to convert listpack to ziplist encoding

Reply:
to run the unit test with flag false I need need first to fix the issue with encoding of big list entry with the ziplist encoding